python/cac_tripplanner/templates/home.html

```diff
@@ -22,14 +22,24 @@ <h1 class="text-center">Eco-friendly exploration of Greater Philadelphia on <em>
             <input id="exploreOrigin" class="typeahead" type="search"
                 data-typeahead-key="origin" placeholder="City Hall" />
             <span class="text">using</span>
-            <select id="exploreMode" class="selectpicker">
-                <option value="TRANSIT,WALK" selected>Transit</option>
-                <option value="BICYCLE">Bike</option>
-                <option value="BICYCLE,TRANSIT">Bike + Transit</option>
-                <option value="WALK">Walk Only</option>
-                <option value="BUSISH,WALK">Transit (Prefer Bus)</option>
-                <option value="TRAINISH,WALK">Transit (Prefer Trains)</option>
-            </select>
+            <div id="directionsMode" class="btn-toolbar transit-types-container">
```

Comment:
I believe we will need to keep the `selectpicker` class set on all inputs for automatic re-querying to continue to work.